scripts/README.md

```diff
@@ -1,3 +1,236 @@
 # Scripts
 
-These scripts rely in a `CDCTI_HOME` environment variable that needs to be set in your environment to your local path to the CDC-TI codebase. You may update and run the `start-here.sh` script to set it. You may also want to add it to your shell's startup file to persist it.
+## Setup
+
+Follow the instructions below to load the environments variables required for these scripts
+
+1. Copy `.env.template` to `.env`
+    ```
+    cp .env.template .env
+    ```
+2. Edit `.env` and make sure to update at least:
+   - `CDCTI_HOME`: local path to the `trusted-intermediary` codebase
+   - `RS_HOME`: local path to the `prime-reportstream` codebase
+   - **Note**: if you don't set `CDCTI_HOME`, none of these scripts will work. Also, please use `$HOME` or the full path to your home directory instead of `~`
+3. Export the environment variables in `.env` by running
+   ```
+   set -a; source .env; set +a
+   ```
+   **Note**: you may also want to add it to your shell's startup file so you don't need to run it for every terminal session.
+4. Run your script
+
+## Available Scripts
+
+### submit.sh
+
+Sends a HL7 message to RS and tracks its status throughout the flow until final delivery. When running locally, it grabs the snapshots of the file in azurite after converting to FHIR, after applying transformations in TI, and after converting back to HL7; and it copies those files to the same folder where the submitted file is. If running in a deployed environment we currently don't have a way to download the files from Azure, but the script will print the relative path for the files in the blob storage container.
+
+#### Requirements
+
+- hurl
```

Comment:
Might be helpful to add links to installation pages for the requirements

Reply:
I added the links to the Resources section at the bottom so we don't have multiple repeated URLs